manifests/pipelines/s3-fetch-pipeline.yaml

```
@@ -318,15 +350,21 @@ spec:
       workspace: build-workspace-pv
   - name: skopeo-copy
     params:
-    - name: srcTLSverify
-      value: "true"
-    - name: destTLSverify
-      value: "true"
+    - name: SOURCE_IMAGE_URL
+      value: ""
+    - name: DESTINATION_IMAGE_URL
+      value: ""
```

Now we can leave this as it was before - just explicitly mentioning TLSverify as true. Just name of the [parameters](https://github.com/openshift-pipelines/tektoncd-catalog/tree/p/tasks/task-skopeo-copy/0.4.2#params) are `SRC_TLS_VERIFY` and `DEST_TLS_VERIFY`.
```suggestion
    - name: SRC_TLS_VERIFY
      value: "true"
    - name: DEST_TLS_VERIFY
      value: "true"
```

---

@MarianMacik Sure. I removed it because these are by default true. But i can add them.